sql/src/main/java/io/crate/planner/projection/MergeCountProjection.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.planner.projection;
+
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableList;
+import io.crate.analyze.symbol.Symbol;
+import io.crate.analyze.symbol.Value;
+import io.crate.metadata.RowGranularity;
+import io.crate.types.DataTypes;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+import java.util.List;
+
+public class MergeCountProjection extends Projection {
+
+    public static final MergeCountProjection PARTIAL_COUNT_AGGREGATION_PROJECTION = new MergeCountProjection();
```

Comment:
I kept it as it was before... Maybe MERGE_COUNT_AGGREGATION_PROJECTION or any other suggestion?